docs/controls/datagrid_guidance/rowdetails.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: How to - Display and Configure Row Details in the DataGrid Control
+title: Display and Configure Row Details in DataGrid

```

Comment:
poor grammar. "in a DataGrid" or "in the DataGrid"

also

You've removed " Control" from this title but not others you've changed here. Is this inconsistency deliberate?